lib/units/websocket/index.js

```diff
@@ -334,6 +336,38 @@ module.exports = function(options) {
         .on('user.settings.reset', function() {
           dbapi.resetUserSettings(user.email)
         })
+        .on('user.keys.accessToken.generate', function(data) {
+          var expiry = Date.now() + 100 * 365 * 24 * 3600
+            , jwt = jwtutil.encode({
+                payload: {
+                  email: user.email
+                , name: user.name
+                }
+              , secret: options.secret
+              , expiry: expiry
+            })
+
+          var tokenId = uuid.v4()
+            , title = data.title
+
+          return dbapi.saveUserAccessToken(user.email, {
```

Comment:
Ok, that's a good point. It's a purely cosmetic issue, but I would appreciate if you could change the format of the token anyway since it would then look more normal :)